config/internal/base/deployment.yaml.tmpl

```diff
@@ -44,7 +44,7 @@ spec:
         app.kubernetes.io/name: modelmesh-controller
         name: {{.ServiceName}}-{{.Name}}
     spec:
-      serviceAccountName: {{.ServiceAccountName}}
+      serviceAccountName: modelmesh-serving-sa
```

Comment:
Is this change one we make, or one from upstream?

Reply:
this is from our side.